docs/guide/queries.md

```diff
@@ -1,25 +1,26 @@
 # Queries
 
-Queries can be written using `strawberry.django.field()` to load the fields defined in the `types.py` file.
+Queries can be written using `strawberry_django.field()` to load the fields defined in the `types.py` file.
 
 ```python
 #schema.py
 
 import strawberry
+import strawberry_django
 
 from .types import Fruit
 
-@strawberry.type
+@strawberry_django.type
 class Query:
 
-    fruit: Fruit = strawberry.django.field()
-    fruits: list[Fruit] = strawberry.django.field()
+    fruit: Fruit = strawberry_django.field()
+    fruits: list[Fruit] = strawberry_django.field()
 
 schema = strawberry.Schema(query=Query)
 ```
 
 !!! tip
 
-    You must name your query class "Query" or decorate it with `@strawberry.type(name="Query")` for the single query default primary filter to work
+    You must name your query class "Query" or decorate it with `@strawberry_type(name="Query")` for the single query default primary filter to work
```

Comment:
Also this. Take a look for any `strawberry_type` to replace with `strawberry.type`, because first one doesn't exist